mobile/src/main/java/ch/epfl/sdp/mobile/application/authentication/AuthenticatedUser.kt

```diff
@@ -60,7 +62,7 @@ class AuthenticatedUser(
    * @param followed the [Profile] to follow.
    */
   suspend fun follow(followed: Profile) {
-    firestore.collection("users").document(followed.uid).update {
+    firestore.collection(ProfileDocument.Collection).document(followed.uid).update {
       arrayUnion("followers", user.uid)
```

Comment:
Missing field.

Reply:
Added in 715a39794e4fd9228eaaf11ad0bf8a1682f5edfd